charts/connect/templates/connect-pdb.yaml

```diff
@@ -9,13 +9,14 @@ metadata:
     {{- include "onepassword-connect.labels" . | nindent 4 }}
 {{- include "onepassword-connect.extraAnnotations" .Values.connect.pdb }}
 spec:
-  {{- with .Values.connect.pdb.maxUnavailable }}
-  maxUnavailable: {{ . }}
-  {{- else }}
+  {{- if .Values.connect.pdb.minAvailable }}
   minAvailable: {{ .Values.connect.pdb.minAvailable }}
   {{- end }}
+  {{- if and .Values.connect.pdb.maxUnavailable (not .Values.connect.pdb.minAvailable)}}
+  maxUnavailable: {{ .Values.connect.pdb.maxUnavailable }}
+  {{- end }}
   selector:
     matchLabels:
       app.kubernetes.io/component: connect
-      {{- include "onepassword-connect.selectorLabels" . | nindent 6 }}
-{{- end }}
\ No newline at end of file
+      app: {{ .Values.connect.applicationName }}
```

Comment:
`[question]`

Just to make sure I fully understand this, we want the Pod Disruption Budget (PDB) to look up for pods that match the specified labels. Is that the case?

Would there be any case in which this should look up for deployments that match a specific set of labels?

Reply:
>Just to make sure I fully understand this, we want the Pod Disruption Budget (PDB) to look up for pods that match the specified labels. Is that the case?

Yes. 

>Would there be any case in which this should look up for deployments that match a specific set of labels?

I don't understand the question. You have two pdb objects, one per deployment in the chart. Each pdb must match the labels for the pods managed by the deployment.

Reply:
> You have two pdb objects, one per deployment in the chart. Each pdb must match the labels for the pods managed by the deployment.

Aha, I see. Therefore there's no scenario in which a PDB would look up for deployments instead of pods, right? If so, then the changes you made make a lot of sense. 😄